projects/angular/src/utils/_mixins.scss

```
@@ -311,7 +311,7 @@ $typography-tokens: (
     // P2, Section Header
     font-size: #{tokens.$cds-alias-typography-secondary-font-size},
     font-weight: #{tokens.$cds-alias-typography-font-weight-medium},
-    line-height: #{tokens.$cds-alias-typography-secondary-line-height},
+    line-height: #{baselinePx(24)},
```

Please use `#{tokens.$cds-global-space-9}` on both places. I don't know why is `baselinePx` function is used previously.

```suggestion
    line-height: #{tokens.$cds-global-space-9},
```